### PR TITLE
additional recipe for tailored build

### PR DESCRIPTION
**Pull request by Alexander-Makaryev**

### CODE REVIEW

File: tailored-recipe/bld.bat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+rem set NO_DIST=1
+
+set DAAL4PY_VERSION=%PKG_VERSION%
+set TBBROOT=%PREFIX%
+set MPIROOT=%PREFIX%\Library
+set DAALROOT=%PREFIX%
```
Comment (fschlimb) [inline tailored-recipe/bld.bat]: Don't overwrite DAALROOT

File: tailored-recipe/bld.bat [hunk 2]
```diff
@@ -0,0 +1,9 @@
+rem set NO_DIST=1
```
Comment (fschlimb) [inline tailored-recipe/bld.bat]: why in comment?

File: tailored-recipe/build.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+if [ "$PY3K" == "1" ]; then
+    ARGS=""
+else
+    ARGS="--old-and-unmanageable"
+fi
+
+#export NO_DIST=1
```
Comment (fschlimb) [inline tailored-recipe/build.sh]: why in comment?

**Comment by PetrovKP** [comment]
@Alexander-Makaryev  Can I close this PR?

**Pull request closed without merging**